src/networks.json

```diff
@@ -1409,5 +1409,20 @@
     "explorer": "https://visionscan.org",
     "start": 75909,
     "logo": "ipfs://QmXgGmxDAW2Mheum3WX7Q52Zi9hvE17Zp1pVtZbcVdThh4"
+  },
+  "12321": {
+    "key": "12321",
+    "name": "Wennect Testnet",
+    "shortName": "Wennect",
+    "testnet": true,
+    "chainId": 12321,
+    "network": "testnet",
+    "multicall": "0x009c55698516Fe3C58105Fe9bBC5A33ECE7A92e4",
```

Comment:
Make sure this contract is verified on the explorer
https://explorer.testnet.wennect.com/address/0x009c55698516Fe3C58105Fe9bBC5A33ECE7A92e4/contracts

Reply:
@ChaituVR now it's verified